### PR TITLE
Enable linter tests

### PR DESCRIPTION
**Pull request by Blast545**
As the title says. Also a minor style fix to pass the `lint_cmake` test.

Signed-off-by: Jorge Perez <jjperez@ekumenlabs.com>

### CODE REVIEW

**Comment by Blast545** [comment]
CI:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/10665/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/6089/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/8676/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/10569/)

**Comment by Blast545** [comment]
Using `ament_lint_auto` and `ament_lint_common` is skipping `ament_copyright` tests, reformated the code to include the tests explicitly.

**Comment by Blast545** [comment]
CI:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/10670/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/6090/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/8677/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/10570/)

**Review by brawner — COMMENTED** [review]
Switch back to auto, I think it was doing the right thing. There are no source files, so the copyright doesn't need to be checked. It did run lint_cmake and xmllint automatically.

https://ci.ros2.org/job/ci_linux/10665/testReport/(root)/projectroot/

**Comment by Blast545** [comment]
This is related to the issue I opened in [ament_lint#237](https://github.com/ament/ament_lint/issues/237): Yes, there shouldn't be a check for source files, but there should be a test checking if the repo has a LICENSE file and a CONTRIBUTING.md one.

As probably dealing with that issue is not a priority now, this PR should do for now. Same as [spdlog_vendor#11](https://github.com/ros2/spdlog_vendor/pull/11)

**Comment by brawner** [comment]
Ok cool, sounds reasonable to me. Add a comment in the CMakeLists pointing to the issue you opened so it can be resolved in the future.

File: CMakeLists.txt [hunk 1]
```diff
@@ -73,5 +73,21 @@ if(NOT console_bridge_FOUND OR "${console_bridge_VERSION}" VERSION_LESS 0.4.1)
   build_console_bridge()
 endif()
 
+# TODO(blast545): This section and package.xml should be updated once copyright auto is solved. See: https://github.com/ament/ament_lint/issues/237
```
Comment (brawner) [inline CMakeLists.txt]:
```suggestion
# TODO(blast545): This section and package.xml should be updated once copyright auto is solved.
# See: https://github.com/ament/ament_lint/issues/237
```

Will it hurt anything if you add ament_lint_auto here too?

**Comment by Blast545** [comment]
Based on your last suggestion @brawner I added `ament_auto` dependencies and the extra `ament_copyright` missing test instead of adding explicitly all the tests.

**Review by brawner — APPROVED** [review]

**Pull request merged**